src/styles/pages/store/cart/index.module.scss

```diff
@@ -142,18 +146,18 @@
         flex-direction: column;
         gap: 0.5rem;
         justify-content: space-around;
-        padding: 0.5rem 0.625rem;
-  
+        padding: 1rem;
+
         > div {
           align-items: baseline;
           display: flex;
           justify-content: space-between;
-  
+
           span {
             text-align: end;
           }
         }
-  
+
         hr {
           border: 0.0625rem solid var(--theme-accent-line-1-transparent);
```

Comment:
and this `<hr>`

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/418a82a3-0208-4d8b-b7b6-5330faa8e1c2)